airbyte-server/src/main/java/io/airbyte/server/converters/ConfigFileArchiver.java

```diff
@@ -72,7 +72,7 @@ public ConfigFileArchiver(final ConfigRepository configRepository) {
   }
 
   public void exportConfigsToArchive(final Path storageRoot) throws ConfigNotFoundException, IOException, JsonValidationException {
-    final StandardWorkspace standardWorkspace = configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID);
+    final StandardWorkspace standardWorkspace = configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID, false);
     if (standardWorkspace != null)
       writeConfigsToArchive(storageRoot, ConfigSchema.STANDARD_WORKSPACE, List.of(standardWorkspace));
     else
```

Comment:
```suggestion
    writeConfigsToArchive(storageRoot, ConfigSchema.STANDARD_WORKSPACE, configRepository.listStandardWorkspaces(true));
```

I think we can get rid of lines 75-79 and just replace it with this. we just want to do the list of all workspaces (regardless of whether they've been deleted).